src/julia-parser.scm

```diff
@@ -1241,7 +1241,18 @@
              (if (ts:space? s)
                  (error (string "space not allowed before \"" t "\"")))
              (take-token s)
-             (loop (list t ex)))
+             (let* ((port (ts:port s))
```

Comment:
This should be handled in the lexer, by removing `'` from the no-suffix list, then the parser test needs to be changed to use a `SuffSet`.

Reply:
Won't this cause any issues with character literals? Or is that handled differently?

Reply:
I think we could use `read-operator` directly for this though.

Reply:
> This should be handled in the lexer, by removing ' from the no-suffix list, then the parser test needs to be changed to use a SuffSet.

When I remove `ctrans-op` form the no-suffix list, `'ᵀ'` parses as `(|'| |'ᵀ|)`, so there needs to be an additional argument to `read-operator` when parsing a call chain vs when starting to parse a new atom.

Reply:
Oops, you're right.